quiz.js

```diff
@@ -192,15 +181,22 @@ function startQuiz() {
 
 function showResultOnTimeOut() {
   let titleText = 'Skończył się czas. Koniec Quizu!';
+  renderResultText(titleText);
+  manageButtonsDisplay();
+  nextButton.style.display = 'none';
```

Comment:
to też można przenieść do manageButtonsDisplay(), w. żadnym z zakończeń nie pokazujemy tego guzika

Reply:
poprawione